netbench-cdk/lib/netbench.ts

```diff
@@ -166,4 +168,35 @@ export class NetbenchInfra extends cdk.Stack {
         return netbenchBucket;
     };
 
+    private createMonitorLambda(): any {
+        const monitorLambda = new cdk.aws_lambda.Function(this, "netbenchMonitor", {
+            runtime: cdk.aws_lambda.Runtime.PYTHON_3_12,
+            handler: "index.lambda_handler",
+            code: cdk.aws_lambda.Code.fromInline(readFileSync('./netbench-monitor/handler.py', 'utf-8')),
+            timeout: cdk.Duration.seconds(15)
```

Comment:
It shouldn't but its the cloud so wondering if its worth bumping this higher?

Reply:
it's never exceeded 7sec in testing, so I was roughly doubling that.